include/install-vitasdk.sh

```diff
@@ -22,7 +22,13 @@ install_vitasdk () {
         sudo mkdir -p $INSTALLDIR
         sudo chown $USER:$(id -gn $USER) $INSTALLDIR
       fi
-      wget -O- "$(get_download_link master linux)" | tar xj -C $INSTALLDIR --strip-components=1
+
+      # Check if architecture is aarch64 (ARM 64-bit)
+      if [ "$(uname -m)" == "aarch64" ]; then
+        wget -O- "$(get_download_link master linux-arm64)" | tar xj -C $INSTALLDIR --strip-components=1
+      else
+        wget -O- "$(get_download_link master linux)" | tar xj -C $INSTALLDIR --strip-components=1
```

Comment:
I remember that when I first read this patch, I'm worried that a script can download `aarch64` version on linux x86_64

so, safe way is, we should split type `linux-arm64` and `linux-x86_64`.
but for backward compatibility, it should be.. `linux-v2` (cuz a tag would be `master-linux-v2.4321`)

```suggestion
        wget -O- "$(get_download_link master linux-v2)" | tar xj -C $INSTALLDIR --strip-components=1
```